velox/type/Filter.h

```diff
@@ -746,6 +750,16 @@ class BigintValuesUsingBitmask final : public Filter {
 
   std::unique_ptr<Filter> mergeWith(const Filter* other) const final;
 
+  std::vector<int64_t> values() const {
+    std::vector<int64_t> values;
```

Comment:
use values.reserve(bitmask_.size()) to avoid re-allocating memory and copying data

Reply:
Masha, this new values vector can be smaller than bitmask_.size(), as not all bitmask bits are true. Is it okay to reserve more than eventually needed? Just to get the language knowledge.

Reply:
It is a trade off. Reserving more memory than needed wastes memory, but not reserving causes extra memory allocations and copy. For example, std::vector would first allocate memory for 1 value, then 2, then 4, then 8, etc. It will copy the data every time it needs to grow the vector.

Reply:
One option is to store the number of values in the filter.

Reply:
Got it. Thank you for explaining!